queries.sql

```diff
@@ -7,4 +7,26 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.5 AND 17.3;
+UPDATE animals SET species = 'unspecified';
+BEGIN ;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokimon' WHERE species IS NULL;
+COMMIT ;
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Kindly check the differene between after `ROLLBACK` state and before the `ROLLBACK` 👍 to ensure that the change have been made 😄 

```sql
 BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```